test/helpers/setup.js

```diff
@@ -1,37 +1,32 @@
-import DestinationRequest from 'testcafe-hammerhead/lib/request-pipeline/destination-request'
 import { createTables } from '@pubsweet/db-manager'
 import config from 'config'
-import start from 'pubsweet/src/startup/start'
 import startS3Server from '@elifesciences/component-service-s3/mock'
+import db from 'pubsweet-server/src/db'
+
 // while writing tests, using the following to start the server avoids having to recompile the app
 // import {startServer: start} from 'pubsweet-server'
 
 import replaySetup from './replay-setup'
 
-let server
 let s3rver
-let serverStartAttempt = false
-
-export async function startServer() {
-  console.log('test/helpers/setup.js::startServer')
-  if (!server) {
-    if (serverStartAttempt) {
-      throw new Error(
-        "Attempting to start the server again, but a previous attempt hasn't been executed yet",
-      )
-    }
-    serverStartAttempt = true
-    // increase timeout to wait for webpack compilation
-    DestinationRequest.TIMEOUT = 180 * 1000
-    console.log('test/helpers/setup.js::startServer await start')
-    server = await start()
```

Comment:
The [`Browser tests` Jenkins stage](https://github.com/elifesciences/elife-xpub/blob/develop/Jenkinsfile#L78) needs to be extended with the separate start command for the server